lms/djangoapps/discussion/tests/test_views.py

```diff
@@ -356,61 +356,72 @@ def setUp(self):
         # course is outside the context manager that is verifying the number of queries,
         # and with split mongo, that method ends up querying disabled_xblocks (which is then
         # cached and hence not queried as part of call_single_thread).
-        (ModuleStoreEnum.Type.mongo, 1, 5, 3, 13, 1),
-        (ModuleStoreEnum.Type.mongo, 50, 5, 3, 13, 1),
+        (ModuleStoreEnum.Type.mongo, False, 1, 5, 3, 13, 1),
+        (ModuleStoreEnum.Type.mongo, False, 50, 5, 3, 13, 1),
         # split mongo: 3 queries, regardless of thread response size.
-        (ModuleStoreEnum.Type.split, 1, 3, 3, 12, 1),
-        (ModuleStoreEnum.Type.split, 50, 3, 3, 12, 1),
+        (ModuleStoreEnum.Type.split, False, 1, 3, 3, 12, 1),
+        (ModuleStoreEnum.Type.split, False, 50, 3, 3, 12, 1),
+
+        # Enabling Enterprise integration increases the number of (cached and uncached) SQL queries by 1,
+        # because the presence of the user's consent for the course must be checked.
+        # But there should be no effect on the number of mongo queries made.
```

Comment:
Since the upstream ticket expressed concern about the performance impact, I think it's ok to leave this test change in place.  I had to update the numbers when I added the view wrapper, so it's nice to explain why that is.

But thanks for noticing I didn't have to wrap the whole darn thing.  Fixed with 556f80fd3f.